tests/src/main/java/com/hazelcast/stabilizer/tests/synthetic/SyntheticTest.java

```diff
@@ -207,18 +211,18 @@ public void doRun() throws Exception {
             operations.addAndGet(iteration % performanceUpdateFrequency);
         }
 
-        private ICompletableFuture invoke(int partitionId) throws Exception {
-            ICompletableFuture f;
+        private ICompletableFuture<SyntheticRequest> invoke(int partitionId) throws Exception {
```

Comment:
Ah, sorry, `Object` should be the better fit. It's `Object` during runtime anyway. Just wanted to get rid of this "unchecked" compiler warning, so we need the future to have a type.